crates/wallet/src/wallet/mod.rs

```diff
@@ -70,7 +70,7 @@ use crate::types::*;
 use crate::wallet::{
     coin_selection::{
         DefaultCoinSelectionAlgorithm,
-        Excess::{self, Change, NoChange},
+        Excess::{Change, NoChange},
```

Comment:
style nit: Only import the enum and use the explicit `Excess::Change` in the remaining code.

```suggestion
        Excess,
```